### ISSUE TITLE
Error - createClient is not a function

### ISSUE THREAD

**Issue by Meenakshi14** [KoteiIto/node-athena#19]
Hi, on trying code I am getting the following error - createClient is not a function.

**Comment by syntacticsolutions** [comment]
What are you passing to the function? What code did you use to get the error?

**Comment by isseu** [comment]
I am using es6 and trying
```node
import athena from 'athena-client';
const client = athena.createClient(clientConfig, awsConfig);
```
I get the same error. I solve it using:
```node
import { createClient } from 'athena-client';
const client = createClient(clientConfig, awsConfig);
```
Buy it should be changed. This is because there is no default export

**Comment by syntacticsolutions** [comment]
use require instead of import.


`const Athena = require('athena-client')

var client = Athena.createClient(clientConfig, awsConfig)


On Jan 17, 2018 7:27 AM, "Enrique Correa" <notifications@github.com> wrote:

I am using es6 and trying

import athena from 'athena-client';const client =
athena.createClient(clientConfig, awsConfig);

I get the same error. I solve it using:

import { createClient } from 'athena-client';const client =
createClient(clientConfig, awsConfig);

Buy it should be changed. This is because there is no default export

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub
<https://github.com/KoteiIto/node-athena/issues/19#issuecomment-358339402>,
or mute the thread

**Comment by KoteiIto** [comment]
Fixed it in version 2.2.0.
Thank you.
